packages/vscode-extension/src/extension.mts

```diff
@@ -46,8 +46,11 @@ export async function activate(context: vsc.ExtensionContext) {
 		{ language: 'json', pattern: '**/data/*/*/**/*.json' },
 	]
 
+	const gameVersion = vsc.workspace.getConfiguration('spyglassmc.env').get('gameVersion')
+
 	const initializationOptions: server.CustomInitializationOptions = {
 		inDevelopmentMode: context.extensionMode === vsc.ExtensionMode.Development,
+		gameVersion: typeof gameVersion === 'string' ? gameVersion : undefined,
```

Comment:
my thought was to maybe raise an error if `typeof gameVersion === 'string'` was false, but i guess vscode will just display an error in the settings.json for us instead?

Reply:
Yes, it will show a warning in the settings file, so the extension should gracefully ignore it.
![image](https://github.com/SpyglassMC/Spyglass/assets/17352009/eee1473e-bd82-42a5-b5b9-55433670e091)